### PR TITLE
Anti-Powercreep Act of 2019 2: Electric Boogaloo

### PR DESCRIPTION
**Pull request by Karl-Franz-I**
## Description
Cuts most of the elitist and snowflake roles with extremely strong roundstart gear from the factions.
Restructures loadouts of most remaining faction roles, changes a few values of items and weapons detailed in the changelog.

## Motivation and Context
After two+ months of absence, claims of powercreep went from "Ok retard fake news" to an indisputable fact that factions have been turned into something much more powerful than ever intended for this backwater part of the wasteland, making us more TDM than ever before. A universal hatred of scouts, the addition of almost mirrored legion roles, and multiple changes to weaponry are a few of the issues this attempts to solve. Currently the single thing a faction has to do now is farm healing items, once that single requirement is met there is no longer any need for scavenging. Before the snowflakes have a meltdown, this is in no way a 100% permanent removal of things like salvaged PA. In the future after further discussion additional loadouts may come into play, and/or there might be other methods implemented to give elite roles and equipment through work that makes your faction deserve such roles being sent as reinforcements. After implementation and the effects can garner some community feedback, values of some armor will be changed if it becomes important. Technical details can be seen/read in the screenshots topic.

## How Has This Been Tested?
Local testing

## Screenshots (if appropriate):
Loadout
NCR
Legion

## Changelog (necessary)
:cl:
add: Added a large amount of loadout variety for most non-recruit roles.
del: Removed all the terrible snowflake jobs.
balance: Lowered slowdown of shields/spears, increased fire delay of the hunting rifle, increased m1911 damage slightly because it was literally worse in every way to the 9mm pistol.
/:cl:

### CODE REVIEW

**Comment by Anton-Kr** [comment]
>thermal lance for cent
Not lore friendly AND OP as fuck imho.

**Comment by Aurum-Dust** [comment]
https://vignette.wikia.nocookie.net/fallout/images/5/5d/Centurion.png/revision/latest?cb=20170913151844

**Comment by Anton-Kr** [comment]
Please disregard I am fucking stupid

**Comment by Aurum-Dust** [comment]
You're fine. The thermal lance _is_ strong, but imo on par with the power fist.  Consider how unwieldy It is to tote around. It takes both hands, and can only fit on your back.

**Comment by Cactus2388** [comment]
Doesn't the Legion believe relying on ranged combat is dishonorable? Giving a Centurion an AMR is kinda gay

**Comment by Karl-Franz-I** [comment]
> Doesn't the Legion believe relying on ranged combat is dishonorable? Giving a Centurion an AMR is kinda gay

I would advise you either do an NCR fallout playthrough, fight in the 2nd hoover dam battle, just play New Vegas in general, or at least look at the centurion wiki where it lists possible equipment spawns. Also the AMR isn't even great compared to the other two options.

**Comment by Cactus2388** [comment]
Alright I actually took at look at the PR

removing MO is dumb fuck off either remove follower too or add it back
NCR has 4 combat roles while Legion has 6
Legion is way better outfitted gear wise
the more loadouts the more balancing that'll eventually need to happen
AMR Centurion is still cringe, fuck you.
Medic trooper loadout doesn't need to exist and will never be picked 
I'll call you a dumbass when it all goes to shit like a week into affect

**Comment by Karl-Franz-I** [comment]
Curious as to where your 4 and 6 lie comes from, if anything it would be 5 and 6 because of the captain, yet gear wise the captain gets the best handgun currently given and can make ammo boxes roundstart and gets a knife thats literally a machete but small instead of normal sized and gets the second best armor. The Lieutenant is and always has been essentially a "raid leader" as well. Even if this argument were true you would be arguing against the organization of the factions themselves.
Legion is outfitted with only a few roles having access to decent weaponry, the Cent and Decanus are really the only threat at range if they choose that loadout (which neuters them entirely in melee), unless you consider Cowboy repeaters to be better than R82s and Service rifles. The legion will have to focus on melee now than before, they dont get mag dumping .44 ez downs at prime level. The more veterans that go explorer, the worse the legion is overall at their melee. So no, the Legion isnt outfitted better except for literally the Centurion. 
The fact you see the AMR as better than the other loadouts when it isnt even up to date with the scout sniper speaks for itself, I hope me calling out your obvious lack of game knowledge you tried to cite isn't the reason you are sperging.
Medic trooper loadout is literally on par with MO save for the health hud which is tech given to literally no one, the MO job was a waste and the only people who say otherwise are the 4 people who played it, the only reason to play MO was chemistry and you can do that as medic so if you don't think people will choose it then they're retards.
I don't think you understand the entire point of this is to limit factions across the board to a lesser overall presence, weaker at roundstart, and less bloat in the role menu, TDM brain. Of course there are a few members of the community who will object to getting their ego chopped off after they took powercreep pills for 3 months. To summarize: fuck your scouts and your heavies mirrored faction bullshit, the NCR officers get plenty of equipment for combat, the MO was a mistake, the AMR is a noob trap, go ahead and look at more code than just this PR before you say stupid shit.

**Comment by JTGSZ** [comment]
I laughed when I saw the original PR that had all of that stuff. It seemed unnecessary considering most of the differences in-factions is just equipment with them not exactly adding to the structure in a good way, the amount of roles doesn't exactly matter. Just the total number and sum you have of whats there.

File: _maps/map_files/Pahrump/Pahrump.dmm [hunk 1]
```diff
@@ -4500,7 +4500,7 @@
 "hru" = (/obj/structure/grille,/obj/structure/window/reinforced/fulltile,/obj/machinery/door/poddoor/preopen{id = "outerkebab_window"},/turf/open/floor/f13{tag = "icon-floorrusty"; icon_state = "floorrusty"},/area/f13/city)
 "huD" = (/obj/effect/spawner/lootdrop/f13/foodspawner,/turf/open/indestructible/ground/outside/desert,/area/f13/wasteland)
 "hvP" = (/obj/machinery/mineral/processing_unit_console,/turf/closed/wall/r_wall/rust,/area/f13/bunker)
-"hwH" = (/obj/effect/landmark/start/f13/vetdecanus,/turf/open/floor/plasteel/f13{icon_state = "plating"},/area/f13/legion)
+"hwH" = (/obj/effect/landmark/start/f13/decanus,/turf/open/floor/plasteel/f13{icon_state = "plating"},/area/f13/legion)
```
Comment (JTGSZ) [inline _maps/map_files/Pahrump/Pahrump.dmm]: Also your travis failed because the landmarks for the removed jobs is still in the .dmm

**Comment by driecg36** [comment]
Couple of issues I'd like to bring up:

1. I still think scouts should be separate roles for each factions. Instead of letting veterans/cpls pick to be explorers, which can lead to a lot of bullshit IMO (like for example if veterans pick explorer class they can just give their old vet armor to someone else), just keep scout roles. One extra role with one or two slots for each faction causes minimal clutter but ensures that there are still dedicated "scout teams." Give them both weaker armor and scoped hunting rifles or something so they aren't powerhouses like current NCR scouts or old explorers and bing bang boom. 

2. I don't like merging decanii into a single role. Even in NV there's a big distinction between the different ranks of decanii and while I appreciate the effort to remove clutter I don't think it's worth changing the leadership structure of the legion. Plus its hardly powercreep considering prime and recruit decanii get weak armor and weak weapons. 

Otherwise I am 300% in favor for this. We can readd the snowflake roles later in not-roundstart capabilites but giving everyone LMGs, assault carbines, heavy rifles, brush guns etc was cancer.

**Comment by Karl-Franz-I** [comment]
1. For corporals/vets, they get the armor distributed to them based on the loadout they choose. Like for your example, they can't give their spare corporal/veteran armor to someone else because they don't get a spare. Also at least in the Legion's case, having it be tied to the veteran job is interesting because the more people who choose scoped rifles the less power the Legion will have in close range/melee due to not having a fireaxe/trench gun wombo combo.
2. The fact of the matter is both sides are cut to 20 combat roles total (As in only 20 players in combat roles on each side in the whole round). Decanus act as squad leaders in charge of a contubernium of legionnaires. The legion would have 2ish contubernium and a centurion. As for them being the same rank, a few suggestions were proposed. The first being that the cent would be able to give one of the two Prime Decanii a veteran helmet, but that may lead to metafriends and favoritism. The other idea I had was to make the vexillarius one of the 3 loadout choices for vet legionnaire and make his job a Veteran Decanus while the other 1-2 can just be primes.

**Comment by driecg36** [comment]
I'm still not convinced about scouts being a loadout choice. I think the roles are distinctive enough in purpose and flavor to warrant being their own thing and that having a set amount of "scouting ability" is reasonable. I also want them to not become the "ranged powerhouse" role for the legion like the explorer used to be and remain a dedicated scouting role (thus the proposal for weaker  armor) 

I do like your idea of having the decanus replacing the vex and making the vex a veteran loadout, it makes a lot of sense. 

Finally, correct me if I'm wrong, but aren't recruit roles "infinite" so to speak? I dislike there being a cap on recruit roles considering they're supposed to be cheap, ill equipped cannon fodder. I am also not crazy about there being such a low set limit for faction roles, as in high pop rounds it will become pretty much impossible to play them unless you join at roundstart.

**Comment by JTGSZ** [comment]
Your travis passed but the last check just failed to complete on travis's part. You can try rerunning it if you want your green checkmark 
Untitled

**Pull request merged**